docs/CommandLine.md

```diff
@@ -0,0 +1,150 @@
+# Command Line
+
+Rokul Runnings has a CLI to execute all commands in the Rokul Runnings class via the command line.
+
+## Setup
+
+### Install
+
+Rokul Runnings CLI can be set up to run either globally or at a project level. In order to use either, they must be installed:
+
+```
+// global
+npm install -g @willowtreeapps/rokul-runnings
+```
+
+```
+// project level
+npm install @willowtreeapps/rokul-runnings
+```
+
+### Defining Config values
+
+Rokul Runnings CLI creates an instance of the Rokul Runnings class based on defaults specified in a config JSON file. The structure of that file looks like:
+
+```json
+{
+  "ip": "", // IP address of the Roku, defaults to "", which will throw an error
+  "username": "rokudev", // username to access Roku Dev portal, defaults to "rokudev"
+  "password": "password", // password to access the Roku Dev portal, defaults to "password",
+  "options": {
+    // Rokul Runnings specific options
+    "pressDelay": 1000, // time to wait in milliseconds between trying keyPresses, defaults to 1000
```

Comment:
I tend to lean towards more verbose names rather than not, so maybe `keyPressDelay` here. That's just an opinion though, `pressDelay` is still understandable.

Reply:
The names in this specific object (`pressDelay`, `retryDelay`, and `retries`) all correspond to specific variables on the Rokul Runnings class, which have similar names (the delays include `inMillis` on the end.) I can update these to more closely match those, but I'd be hesitant to overhaul the whole thing right now 😄

Reply:
so long as they match, then that's fine! From your comment it seemed like they might be being called 'keyPress' elsewhere

Reply:
Yeah, that's a weird remnant of the translation from Roku's python automation library to this one. _Technically_, a keyPress is both the action of pressing a button (either down, up or both) and also only the action of a down -> up press.